docs/development/extensions-core/s3.md

```diff
@@ -131,6 +131,7 @@ For example, to set the region to 'us-east-1' through system properties:
 |`druid.s3.disableChunkedEncoding`|Disables chunked encoding. See [AWS document](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/AmazonS3Builder.html#disableChunkedEncoding--) for details.|false|
 |`druid.s3.enablePathStyleAccess`|Enables path style access. See [AWS document](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/AmazonS3Builder.html#enablePathStyleAccess--) for details.|false|
 |`druid.s3.forceGlobalBucketAccessEnabled`|Enables global bucket access. See [AWS document](https://docs.aws.amazon.com/AWSJavaSDK/latest/javadoc/com/amazonaws/services/s3/AmazonS3Builder.html#setForceGlobalBucketAccessEnabled-java.lang.Boolean-) for details.|false|
+|`druid.s3.maxConnections`|Max concurrent connections to S3 per task.|None (fallbacks to AWS SDK default)|
```

Comment:
The description makes it seem like this is being set only for tasks.  It's being set globally for anything that interacts with S3.

Reply:
Nit: "fallbacks" --> "falls back"